src/main/java/frc/robot/subsystems/swerve/modules/ModuleConstants.java

```diff
@@ -1,20 +1,35 @@
 package frc.robot.subsystems.swerve.modules;
 
 import edu.wpi.first.math.geometry.Rotation2d;
-import frc.robot.constants.LogPaths;
-import frc.robot.subsystems.swerve.SwerveConstants;
 import frc.utils.Conversions;
 import frc.utils.battery.BatteryUtils;
 
-public record ModuleConstants(double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
+public record ModuleConstants(
+        String modulesLogPath,
```

Comment:
```suggestion
        String logPath,
```
u r in module constants